src/Symfony/Component/ErrorRenderer/Exception/FlattenException.php

```diff
@@ -374,3 +374,6 @@ public function getAsString()
         return rtrim($message);
     }
 }
+
+// to be removed in 5.0
+class_exists(\Symfony\Component\Debug\Exception\FlattenException::class);
```

Comment:
let's inline instead, also we don't need the legacy class to be autoloadable, isn't it?

Reply:
I've reverted the logic